src/helpers/buffers/ordering_mpsc.rs

```diff
@@ -324,6 +358,7 @@ mod unit {
         },
     };
     use futures::{future::join, FutureExt};
+    use futures_util::StreamExt;
```

Comment:
futures_util::StreamExt or futures::stream::StreamExt ?  Or are these aliases?

Reply:
looks like an alias to me https://docs.rs/futures-util/latest/futures_util/#reexports, but it is worth importing it from `futures` crate for consistency. My Intellij makes it too easy to import without paying attention :(